README.md

```diff
@@ -45,6 +45,13 @@ The current version of Livepeer requires [ffmpeg](https://www.ffmpeg.org/).
 ### geth
 Livepeer requires a local Ethereum node. To set it up, follow the [Ethereum Installation Guide](https://github.com/ethereum/go-ethereum/wiki/Building-Ethereum)  (We have tested with 1.6.7)
 
+### Creating Etherium Account
```

Comment:
Is this step necessary @ericxtang? I don't believe I've had to do it when setting up local nodes to sync with our private testnet?

Reply:
I was unable to figure out which account is being used. so I thought if a user creates an account, he will have a better understanding.

Reply:
Yeah I think it'll be useful if the user hasn't generated an account.  @r-a-x, there is a mis-spelling "Ethereum" instead of "Etherium".

Reply:
@ericxtang should I correct it and generate an another pull request ?